FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -5,20 +5,28 @@
 import Foundation
 
 public final class RemoteFeedLoader: FeedLoader {
-	private let url: URL
-	private let client: HTTPClient
-
 	public enum Error: Swift.Error {
 		case connectivity
 		case invalidData
 	}
 
+	private let url: URL
+	private let client: HTTPClient
+
 	public init(url: URL, client: HTTPClient) {
 		self.url = url
 		self.client = client
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		self.client.get(from: url, completion: { [weak self] result in
```

Comment:
Please remove explicit `self` to follow the codebase convention
```suggestion
		client.get(from: url, completion: { [weak self] result in
```